src/components/molecules/NavBar/components/SoundCloudRadio/SoundCloudRadio.scss

```diff
@@ -0,0 +1,47 @@
+@import "scss/constants";
+
+.SoundCloudRadio {
+  position: relative;
+  padding: font-size--parent(0.5);
+
+  &__button {
+    @include square-size(40px);
+    display: block;
+    background-color: var(--content--under-40a);
+    border-radius: $border-radius--max;
+    margin: 1px $margin-right--navbar-links 1px 1px;
+    cursor: pointer;
+    object-fit: contain;
+
+    background-image: url(/assets/icons/navbar-link-radio.png);
+    background-size: 80px 40px;
+    // NOTE: Currently station is displayed in iframe so no info exists to know when it is on/off
+    // animation: radioanim 2s steps(2) infinite;
```

Comment:
It might not always be this case.

Can just leave it blinking forever, but that's not the same  UI meaning like in the normal radio, so it's left for future reference in case the UX getting unified.